### PR TITLE
Adding gaussian normalization

### PR DESCRIPTION
**Pull request by rosecers**
Not quite sure why it needs the volume, but that's apparently the thing.

### CODE REVIEW

**Review by arthur-lin1027 — COMMENTED** [review]
this is fine, but why the nomenclature change? Are we using the word "types" instead of species in the other files?

File: anisoap/reference/projection_coefficients.py
Comment (arthur-lin1027) [inline anisoap/reference/projection_coefficients.py:1]: good, but why this nomenclature change? is this more consistent with our other files/rascaline?

Comment (rosecers) [reply anisoap/reference/projection_coefficients.py:1]: This is consistent with the new rascaline change. I'd suggest we merge #36 then rebase this branch, fyi

**Review by arthur-lin1027 — APPROVED** [review]
i'm ok with merging this in now for the sake of not leaving this branch dangling for way too long, although this will likely need further iteration.

In the future I will make various tests to ensure gaussian normalization and to test other pieces of the code as well (e.g. that Jigyasa's tensor products do what they're supposed to)

File: anisoap/representations/ellipsoidal_density_projection.py [hunk 1]
```diff
@@ -118,12 +118,24 @@ def pairwise_ellip_expansion(
...
-                    precision, center = radial_basis.compute_gaussian_parameters(
-                        r_ij, lengths, rot
-                    )
-
-                    moments = compute_moments_inefficient_implementation(
-                        precision, center, maxdeg=maxdeg
+                    volume = 4.0 / 3.0 * np.pi * np.product(lengths)
+
+                    length_norm = (
+                        np.product(lengths) * volume * (2.0 * np.pi) ** (3.0 / 2.0)
```
Comment (rosecers) [inline anisoap/representations/ellipsoidal_density_projection.py:124]: As per our GPT friend, we were reminded that the covariance matrix is the product of the squared semiaxes, not just the semiaxes. 

```suggestion
                    length_norm = (
                        np.product(lengths) * np.product(lengths) * (2.0 * np.pi) ** (3.0 / 2.0)
```

**Comment by rosecers** [comment]
@arthur-lin1027 should we close?

File: anisoap/representations/ellipsoidal_density_projection.py [hunk 2]
```diff
@@ -118,12 +118,24 @@ def pairwise_ellip_expansion(
 
                     rot = rotation_matrices[j_global]
                     lengths = ellipsoid_lengths[j_global]
-                    precision, center = radial_basis.compute_gaussian_parameters(
-                        r_ij, lengths, rot
-                    )
-
-                    moments = compute_moments_inefficient_implementation(
-                        precision, center, maxdeg=maxdeg
+                    volume = 4.0 / 3.0 * np.pi * np.product(lengths)
```
Comment (rosecers) [inline anisoap/representations/ellipsoidal_density_projection.py:121]: Following #38, this line will no longer be needed.

**Review by cbefan — APPROVED** [review]
tests work, normalization seems right, very cool!

**Pull request merged**